apps/nextjs-website/src/_contents/webinars.ts

```diff
@@ -19,6 +58,7 @@ export const webinars: readonly Webinar[] = [
       `<p>La riscossione delle sanzioni previste dal Codice della strada è uno dei servizi più diffusi che i Comuni si trovano a dover gestire. L'utilizzo integrato di pagoPA, IO e SEND può rappresentare per le Amministrazioni un fattore chiave per efficientare tutti i processi legati al loro intero ciclo di vita, dall'emissione del preavviso fino all'invio dell'eventuale contravvenzione. Durante l’incontro verranno analizzate tutte le fasi atte all'emissione e riscossione di una multa, rappresentando anche best practice e vantaggi derivanti da un utilizzo consapevole e sinergico delle piattaforma PagoPA. Lo scopo ultimo è di fornire ai Comuni gli strumenti utili per beneficiare appieno di tutti i vantaggi legati alla gestione digitale del servizio.</p>\n`,
     slug: 'PagoPA-multe',
     isVisibleInHome: true,
+    isVisibleInWebinars: true,
```

Comment:
Have you considered calling it `isVisibleInWebinarListPage` or `isVisibleInList`?
```suggestion
    isVisibleInList: true,
```